rn/Teacher/src/modules/assignment-details/components/PublishedIcon.js

```diff
@@ -25,7 +25,7 @@ import { View, Image, StyleSheet } from 'react-native'
 import Images from '../../../images'
 import i18n from 'format-message'
 import { Text } from '../../../common/text'
-import colors from '../../../common/colors'
+import { colors } from '../../../common/stylesheet'
```

Comment:
Assuming we want to keep the updates, should this use createStyleSheet?

Reply:
fixed.